src/ex2/index.html

```diff
@@ -17,8 +17,13 @@ <h1 class="app-name">Todo App</h1>
           <button type="button" id="list-item-submit">+</button>
         </div>
         <ul id="list"></ul>
+        <div class="delete-all">
+        <button type="button" id="delete-all-button">Delete All</button>
+        </div>
       </div>
     </div>
   </body>
-  <script src="main.js"></script>
+  <script type="module" src="main.js"></script>
+  <script type="module" src="ItemManager.js"></script>
```

Comment:
you dont have to import ItemManager.js and PokemonClient.js as they're imported in main.js

Reply:
fixed!